src/trig/tokens.ts

```diff
@@ -0,0 +1,18 @@
+const { turtleTokenTypes } = require('../turtle/tokens');
+import { TokenType } from 'chevrotain';
+import { turtleTokenMap } from 'turtle/tokens';
+import { sparqlTokenMap } from 'sparql/tokens';
+
+export const trigTokenMap = {
+  LCurly: sparqlTokenMap.LCurly,
+  RCurly: sparqlTokenMap.RCurly,
+  GRAPH: sparqlTokenMap.GRAPH,
+  ...turtleTokenMap,
+};
+
+export const trigTokenTypes: TokenType[] = [
+  sparqlTokenMap.LCurly,
+  sparqlTokenMap.RCurly,
+  sparqlTokenMap.GRAPH,
```

Comment:
The order of these tokens matters, and putting `GRAPH` so high up will result in failures to parse valid TriG/Turtle. Example where this will fail:

```
:something graphPrefix:somethingElse :someObject .
```